### DISCUSSION TITLE
Feature Request: Persistent or save-able filter selections in entry index

### DISCUSSION THREAD

**Discussion by ryanfoote** [craftcms/cms#8712] [Ideas]
### Description

It would be helpful if the entry index sported the ability to either keep the last set of filters (entry status, search term, etc.) persistent until cleared, and/or if an admin could save a set of filter criteria that they could easily re-apply to the entry index with a single click.

Sample use case: If an admin has filtered down all entries that are currently disabled and is reviewing them one by one, the admin has to repeatedly set the "status" filter to disabled each time they return to the entry index to select the next entry.

**Comment by TonyDeStefano** [comment]
+1

**Comment by brandonkelly** [comment]
Kinda similar to #1630
